lib/tests/testscripts/testscript_engine.rb

```diff
@@ -2,18 +2,19 @@ module Crucible
   module Tests
     class TestScriptEngine
 
+      @@scripts = nil
+
       def initialize(client=nil, client2=nil)
         @client = client
         @client2 = client2
-        load_testscripts
       end
 
       def tests
-        @scripts || []
+        @@scripts || []
```

Comment:
Why not just initialize `@@scripts` to `[]` rather than `nil`? That way you wouldn't have to do these `||`s with `[]`